File: B/1391B.py
def minChange() -> int:
    n, m = map(int,I().split())
    con_belt = [list(I().strip()) for _ in range(n)]
    
    min_change = 0
    for _i in range(n):
        for _j in range (m):
            ch = con_belt[_i][_j]
            if (( _j == m - 1 and ch == 'R') or ( _i == n - 1 and ch == 'D')):
                min_change+=1
    return min_change

# test case run
I = input
t = int(I())
for i in range(t):
    print(minChange())
    

File: 1001-1500/1391B.cpp
#include <bits/stdc++.h>
using namespace std;

int minChange(){
    int n, m; cin >> n >> m;
    char con_belt[n][m];
    int min_change = 0;
    for (int i = 0; i < n; i++){
        for (int j = 0; j < m; j++) {
            char ch; cin >> ch;
            con_belt[i][j] = ch;
            if ( j == m - 1 && ch == 'R') min_change++;
            if ( i == n - 1 && ch == 'D') min_change++;
        }
    }
    return min_change;
}

//main function
int main() {
    int t; cin >> t;
    while (t--) cout << minChange() << "\n";
    return 0;
}